packages/terra-application-header/src/ApplicationHeader.scss

```diff
@@ -0,0 +1,26 @@
+:local {
```

Comment:
Should the colors in this file be themeable?

Reply:
Yes, we'll want them to be themeable for the legacy app. Looks like all the colors in this file are now using CSS custom properties.